spec/features/carrier_spec.rb

```diff
@@ -21,6 +53,20 @@
     expect(page).to have_content('test model')
   end
 
+  scenario 'NEW' do
+    visit new_carrier_path
+    fill_in 'Name', with: 'Test Name'
+    fill_in 'Item', with: 9
+    fill_in 'Manufacturer', with: 'Test Manufacturer'
+    fill_in 'Model', with: 'Test Model'
+    fill_in 'Color', with: 'White'
+    find('#carrier_category_id').find(:option, category.name).select_option
+
+    click_on 'Create Carrier'
+
+    expect(page).to have_content('Carrier successfully created')
```

Comment:
Thanks for doing all the changes! One thing to note, your NEW and CREATE action tests look the same. What I'd advise to do:
- the 'NEW' tests should just cover landing on the 'new_carrier_path' path, not submitting the form
- the 'CREATE' tests can fill out the form and submit it.

Reply:
Thanks @DamirSvrtan. I thought the same but I didnt know what to put on the new test, what should I expect to see? once an admin goes to the /new, he/she should see a form but how can I test this? I want to do the update.

Reply:
Nevermind, I will just check for the "New Carrier" content.

Reply:
done @DamirSvrtan

Reply:
Hey @samanthakem, yeah, just checking for the 'New Carrier' is great!